src/modules/asab-config/index.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import Module from 'asab-webui/abc/Module';
 import ConfigContainer from "./ConfigContainer";
 import MicroservicesContainer from "./MicroservicesContainer";
-
+import AboutScreen from '../about/AboutScreen';
```

Comment:
@t-pyrope This is wrong, you dont want to import `About` screen to the `asab-config` module, please remove your implemented code from this `src/modules/asab-config/index.js` file